drivers/exec/driver.go

```diff
@@ -227,6 +245,12 @@ func (tc *TaskConfig) validate() error {
 		return fmt.Errorf("cap_drop configured with capabilities not supported by system: %s", badDrops)
 	}
 
+	usernameToLookup := getUsername(cfg)
+
+	if err := validators.UserInRange(users.Lookup, usernameToLookup, driverConfig.DeniedHostUids, driverConfig.DeniedHostGids); err != nil {
+		return err
+	}
+
```

Comment:
I don't think this is the right place for this validation - we've now added 2 new arguments to pass in on this `TaskConfig.validate()` method, and then ignored the `tc` itself - suggesting we should leave this method alone and do the assertion elsewhere.

Reply:
Yeah the additional args felt bad to me, but this does feel like a validation step 🤔 - I'll see what I can do to make it nicer